continuous-integration/linux/create-sdist.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR=$(realpath "$SCRIPT_DIR/../..")
+
+# Install minimal requirements to create the source distributions
+pip install --requirement "$SCRIPT_DIR/../python-requirements/build.txt" || exit $?
+
+# Create source distribution
+(cd $ROOT_DIR && python setup.py sdist) || exit $?
```

Comment:
Can this line be split as well?

Reply:
I'd like to avoid that. The only reason why I do "cd" is that setup.py must be called from the same directory as the file itself, and I don't need the directory change to live beyond this line. The benefit of doing it this way is that the directory change is done in a subshell, so the working directory is unchanged as the script moves on.

Reply:
Perhaps a comment might be needed? I did not think of this when I read this part of the code.

Reply:
Sure I can add a comment.